### ISSUE TITLE
tmkms hangs unexpectedly

### ISSUE THREAD

**Issue by Tomas-Eminger** [iqlusioninc/tmkms#481]
I am running tmkms for multiple networks and it hangs unexpectedly after many days of running.

```
Feb 14 19:25:22 kms tmkms-mainnet[834893]: 2022-02-14T18:25:22.173053Z  INFO tmkms::session: [sommelier-3@tcp://10.137.18.10:26658] signed PreCommit:<nil> at h/r/s 2267640/0/2 (103 ms)
Feb 14 19:25:25 kms tmkms-mainnet[834893]: 2022-02-14T18:25:25.540179Z  INFO tmkms::session: [agoric-3@tcp://10.137.11.10:26658] signed PreVote:<nil> at h/r/s 3750935/0/1 (103 ms)
Feb 14 19:25:25 kms tmkms-mainnet[834893]: 2022-02-14T18:25:25.913424Z  INFO tmkms::session: [cosmoshub-4@tcp://10.137.16.10:26658] signed PreVote:E6DA6D20AB at h/r/s 9427943/0/1 (126 ms)
Feb 14 19:25:26 kms tmkms-mainnet[834893]: 2022-02-14T18:25:26.017201Z  INFO tmkms::session: [agoric-3@tcp://10.137.11.10:26658] signed PreCommit:<nil> at h/r/s 3750935/0/2 (137 ms)
Feb 14 19:25:26 kms tmkms-mainnet[834893]: 2022-02-14T18:25:26.551435Z  INFO tmkms::session: [cosmoshub-4@tcp://10.137.16.10:26658] signed PreCommit:E6DA6D20AB at h/r/s 9427943/0/2 (126 ms)
Feb 14 19:25:27 kms tmkms-mainnet[834893]: 2022-02-14T18:25:27.366235Z  INFO tmkms::session: [sommelier-3@tcp://10.137.18.10:26658] signed PreVote:73424E7EDA at h/r/s 2267641/0/1 (126 ms)
Feb 14 19:25:27 kms tmkms-mainnet[834893]: 2022-02-14T18:25:27.499522Z  INFO tmkms::session: [sommelier-3@tcp://10.137.18.10:26658] signed PreCommit:73424E7EDA at h/r/s 2267641/0/2 (126 ms)
```

There is nothing else in the log, after restart everything runs smoothly.

Version: tmkms 0.11.0-pre

**Comment by tony-iqlusion** [comment]
Are you using a YubiHSM2?

Can you try the final v0.11 release?

See also: #37

**Comment by tony-iqlusion** [comment]
Closing this in favor of #37. Note that the fix shipped in v0.11.0-pre.2
